src/SFA.DAS.EmployerAccounts.Web/Extensions/UrlHelperExtensions.cs

```diff
@@ -94,6 +103,27 @@ private static string AccountAction(UrlHelper helper, string baseUrl, string pat
             return Action(baseUrl, accountPath);
         }
 
+        private static string CommitmentAction(UrlHelper helper, string baseUrl, string routeName)
+        {            
+            var hashedAccountId = helper.RequestContext.RouteData.Values[ControllerConstants.AccountHashedIdRouteKeyName];
```

Comment:
Again this logic should definitely not be in here as it assumes the viewModel

Reply:
Done